### ISSUE TITLE
app list didn't updated properly

### ISSUE THREAD

**Issue by GreenMystic** [markusfisch/PieLauncher#137]
disabling apps something isn't hidding app on app list 🗿
trigger is simple:
try to disable or enable apps periodically
it won't updated on launcher oftentimes 

can you pls fix it?
![Screenshot_20241019-082825](https://github.com/user-attachments/assets/0160341f-3bbe-4ad6-8a59-e81e742a4803)
![Screenshot_20241019-082905](https://github.com/user-attachments/assets/cf104073-05f2-405c-883e-caa6b0a0e9fd)

thank you

**Comment by markusfisch** [comment]
Since all the apps are reindexed after hiding or showing an app, it may take a little while until the change takes effect. Sorry for this limitation, I will change this in future versions.

**Comment by markusfisch** [comment]
Yes, this should be fixed with https://github.com/markusfisch/PieLauncher/commit/edd4ee6f5d86bbc47be0182d137b8f484cb8f295

A new version will become available soon!
